engine/src/index.d.ts

```diff
@@ -2355,6 +2406,20 @@ export class WWTControl {
    */
   renderType: ImageSetType;
 
+  /** Get the name of the reference frame associated with the current view.
+   *
+   * The current reference frame defines the physical coordinates of the view
+   * and the list of layers that are included in the current rendering process.
+   * The return value of this function can be indexed into
+   * [[LayerManager.get_allMaps]] to find the root [[LayerMap]] that is used to
```

Comment:
`get_allMaps` is more correct here. If you set the view to Mars mode, the current reference frame will be "Mars", which is not a key in the return value of `get_layerMaps()` because Mars comes below Sun in the hierarchy. E.g.:

https://github.com/WorldWideTelescope/wwt-webgl-engine/blob/master/engine/wwtlib/Layers/LayerManager.cs#L799